### DISCUSSION TITLE
Skinshortcuts Question

### DISCUSSION THREAD

**Discussion by xyzfre** [jurialmunkey/skin.arctic.horizon.2#820] [Discussion]
I noticed over the weekend there's a updated version of Skinshortcuts from Mikesilvo164 Should we be using that one or from Kodi Repo my apologies didn't know where else to ask and didn't want to open a issue with a question sorry

**Comment by jurialmunkey** [comment]
I just use the repo version myself

**Reply by xyzfre** [reply]
Thanks @jurialmunkey for your response just wanted to make sure it was compatible with AH2 and thank you for all your AMAZING work and dedication AH2 is the BEST Skin ever 💯💯
